Formula/quilt-installer.rb

```diff
@@ -0,0 +1,24 @@
+class QuiltInstaller < Formula
+  desc "Installer for Quilt for the vanilla launcher"
+  homepage "https://quiltmc.org/"
+  url "https://maven.quiltmc.org/repository/release/org/quiltmc/quilt-installer/0.4.3/quilt-installer-0.4.3.jar"
+  sha256 "045668265303d41dae13abc5eb000e988ad409b792431663e06b46ad0811f80a"
+  license "Apache-2.0"
+
+  livecheck do
+    url "https://meta.quiltmc.org/v3/versions/installer"
+    regex(/["']url["']:\s*["'][^"']*?quilt-installer[._-]v?(\d+(?:\.\d+)+)\.jar/i)
```

Comment:
maybe this? cc @samford 
```suggestion
    regex(/quilt-installer[._-]v?(\d+(?:\.\d+)+)\.j/i)
```

Reply:
We only use a loose extension like `\.t` for tarball files, since there's a lot of variation for those. For others, like jar files, we simply use the full extension (`.jar`). In general, I'm fine with the original regex.

That said, we have some other formulae with a Maven `stable` URL that check the `maven-metadata.xml` file for versions and we have an established pattern for that. Though this doesn't use maven.org, there's something to be said for using that approach, as the [first-party installation page for the client](https://quiltmc.org/install/client) also uses `maven-metadata.xml` to determine the latest version.

I plan to eventually create a `Maven` strategy, so using that approach would allow us to simplify this `livecheck` block in the future along with the others. I'll create a separate suggestion for this in a moment.